packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -200,15 +206,23 @@ function registerCommands(): vscode.Disposable {
     forceOrgDisplayDefaultCmd,
     forceOrgDisplayUsernameCmd,
     forceGenerateFauxClassesCmd,
+    forceProjectCreateCmd,
     internalCancelCommandExecution
   );
 }
 
-export function activate(context: vscode.ExtensionContext) {
+export async function activate(context: vscode.ExtensionContext) {
   console.log('SFDX CLI Extension Activated');
 
   // Context
-  vscode.commands.executeCommand('setContext', 'sfdx:project_opened', true);
+  const sfdxProjectOpened = await vscode.workspace.findFiles(
```

Comment:
Previously always set to true because extension would only activate if sfdx-project.json was present. If the project create command is the activation reason, this may not be true.